OpenDreamRuntime/Procs/Native/DreamProcNativeWorld.cs

```diff
@@ -38,5 +38,67 @@ public static async Task<DreamValue> NativeProc_Export(AsyncNativeProc.State sta
 
             return new DreamValue(list);
         }
+
+        [DreamProc("GetConfig")]
+        [DreamProcParameter("config_set", Type = DreamValue.DreamValueType.String)]
+        [DreamProcParameter("param", Type = DreamValue.DreamValueType.String, DefaultValue = null)]
```

Comment:
`DefaultValue` is null by default. It's only used when the passed argument is null, so it wouldn't make sense anyways.
```suggestion
        [DreamProcParameter("param", Type = DreamValue.DreamValueType.String)]
```